### DISCUSSION TITLE
配置腾讯云函数的时候出现了这个是什么原因？

### DISCUSSION THREAD

**Discussion by hani1233214** [IceTiki/ruoli-sign-optimization#123] [Q&A]
__
![6](https://user-images.githubusercontent.com/115795447/195844348-a233201d-c2d4-418c-9eb2-972e759ffe4a.png)

**Comment by hani1233214** [comment]
加了仓库的验证码代码后就出现这种情况

**Comment by hani1233214** [comment]
我是小白，我是直接复制作者的代码到云函数的，望相关大佬解惑

**Comment by xdddhd** [comment]
### 呃呃，你是使用哪个语言的？是否有简体中文文字认知障碍？
![image](https://user-images.githubusercontent.com/89734693/195861427-20db8bb8-5eee-435d-bc8d-a69350b20ae4.png)

**Comment by xdddhd** [comment]
https://github.com/IceTiki/ruoli-sign-optimization/blob/master/documents/%E7%94%A8%E6%88%B7%E9%85%8D%E7%BD%AE%E9%AB%98%E7%BA%A7%E6%95%99%E7%A8%8B.md

**Comment by Ben11123** [comment]
你查下176行，5列开头缩进

**Comment by hani1233214** [comment]
> 你查下176行，5列开头缩进

![MFC7H(T7@8F1T_00HZ2FHYH](https://user-images.githubusercontent.com/115795447/195876875-d9e9c981-c0c1-449f-8c57-20abde4fcc5a.png)
主要是这个地方出了问题，但目前不知道咋解决

**Comment by hani1233214** [comment]
> https://github.com/IceTiki/ruoli-sign-optimization/blob/master/documents/%E7%94%A8%E6%88%B7%E9%85%8D%E7%BD%AE%E9%AB%98%E7%BA%A7%E6%95%99%E7%A8%8B.md

      : returns dict: 用于更新表单(self.form)的字典(如果不需要验证码返回{}, 如果需要返回)
        '''
        headers = session.headers.copy()
        # ====================检查验证码====================
        have_cap = f"{host}wec-counselor-attendance-apps/student/attendance/checkValidation"
        data = {'deviceId': deviceId}
        headers.update({
            'CpdailyStandAlone': '0',
            'extension': '1',
            'Content-Type': 'application/json; charset=utf-8',})
        res = session.post(
            url=have_cap, data=json.dumps(data), headers=headers)
        res = res.json()
        haveCap_data = res['datas']
        LL.log(1, "检查是否需要填写验证码", haveCap_data)
        if not haveCap_data['validation']: 
            '''如果不需要填写验证码, 则直接返回'''
            return {}

        # ====================获取验证码====================
        headers.update({
            'Content-Type': 'multipart/form-data; boundary=----WebKitFormBoundaryBlRdUZvbYBzP5FaF',
            'deviceId': deviceId,})
        url = f"{host}captcha-open-api/v1/captcha/create/scenesImage"
        data = [
            ("accountKey", haveCap_data['accountKey']),
            ("sceneCode", haveCap_data['sceneCode']),
            ("tenantId", haveCap_data['tenantId']),
            ("userId", haveCap_data['userId'])]

        res = session.post(url=url, data=MultipartEncoder(
            data, boundary="----WebKitFormBoundaryBlRdUZvbYBzP5FaF"), headers=headers)
        capCode = res.json()
        LL.log(1, "获取验证码", capCode)

        # ====================识别验证码====================

        event = {
            "msg": f"请求图片验证码识别",  # 触发消息
            "from": "liteTools.handleCaptcha",  # 触发位置
            "code": 300,}
        answerkey = UserDefined.trigger(
            event, context={"capcode": capCode}, exceptError=False)


        # ====================提交验证码====================
        url = f"{host}captcha-open-api/v1/captcha/validate/scenesImage"
        data = [
            ("accountKey", haveCap_data['accountKey']),
            ("sceneCode", haveCap_data['sceneCode']),
            ("tenantId", haveCap_data['tenantId']),
            ("userId", haveCap_data['userId']),
            ("scenesImageCode", capCode["result"]['code'])]
        data.extend([('scenesImageCodes', i) for i in answerkey])
        res = session.post(url=url, data=MultipartEncoder(
            data, boundary="----WebKitFormBoundaryBlRdUZvbYBzP5FaF"), headers=headers)
        res = res.json()
        LL.log(1, "提交验证码", res)
        return {"ticket": res['result']}


class NT: 
 -991,17 +1058,28 @@ class UserDefined: 
        101: "全局任务结束"
        200: "局部任务开始"
        201: "局部任务结束"
        300: "图形验证码识别"


大佬可以帮我看一下现在哪里还存在问题吗

**Comment by xdddhd** [comment]
你要不把config备份一下，重新部署一个吧

**Comment by IceTiki** [comment]
是config.yml里面有格式错误，别改.py文件
